### PR TITLE
include payment token into query

### PR DESCRIPTION
**Pull request by santiagonoguez**
I added the payment token, based on authKey, and tested in BigQuery. I also tested that it works well with null values:

paymentToken_log_output

### CODE REVIEW

File: src/main/resources/sql/companySnapshot.sql [hunk 1]
```diff
@@ -6,53 +6,54 @@ LANGUAGE js AS """
...
-  C.street, 
+  C.street,
   C.unit,
   C.city,
   C.province as state,
   C.country,
   C.creationDate,
-  C.hubSpotId
+  C.hubSpotId,
+  substr(C.hubSpotId, length(C.hubSpotId) - 5, 6) as paymentToken
```
Comment (alex-deaconu) [inline src/main/resources/sql/companySnapshot.sql:33]: @santiagonoguez you're taking a `substr` of the `hubspotId` here. Should it not be the `authKey`? Thanks.

Comment (santiagonoguez) [reply src/main/resources/sql/companySnapshot.sql:33]: oh my, what a stupid mistake I did, thanks ! :D

**Comment by santiagonoguez** [comment]
@alex-deaconu I fixed the field referenced and tested again the query:

query_updated
